### ISSUE TITLE
World coordinate orientation and gravity direction

### ISSUE THREAD

**Issue by jinlinyi** [facebookresearch/co3d#28]
Could anyone clarify how is the world coordinate oriented? Is there any way to figure out the gravity direction? Thanks!

**Comment by davnov134** [comment]
Hi, the object models have all been centered around and scale-normalized to a unit standard deviation averaged over the 3 coordinate axes. We also aligned the principal components of the point clouds with the axes of the world coordinate system. We did not perform any additional manual alignment of the world coordinate frames. I.e. the gravity direction is not uniquely defined in the dataset.

**Comment by SteveJunGao** [comment]
Thanks for the information!
I'm wondering, in the experiment section of the paper (Sec. 4 & 5), are you using this coordinate system? or do you have other additional alignment to make different shapes  lie in a canonical coordinate system?

**Comment by davnov134** [comment]
Yes all experiments use this kind of alignment. We aimed to avoid any kind of coordinate system alignment since any alignment inaccuracies tend to kill performance. Instead we devised methods that are invariant to the global coordinate frame alignment.

**Comment by bibbygoodwin** [comment]
> We also aligned the principal components of the point clouds with the axes of the world coordinate system
@davnov134 - thank you for this detail. To clarify, I'm assuming this refers only to your experiments, and not to the pointclouds as saved in `.ply` files in the datasets. I believe that these are aligned with respect to the initial view? That is, the orientation of the camera object associated with the first frame of each sequence is the identity, or very very close to it (as confirmed by examining the data). This wouldn't be compatible with the further alignment logic you described? Could you clarify this?
Thank you!
